unit_tests/engine/test_rule_loader.cpp

```diff
@@ -903,4 +903,32 @@ TEST_F(test_falco_engine, list_name_invalid)
 
   ASSERT_FALSE(load_rules(rules_content, "rules.yaml"));
   ASSERT_TRUE(check_error_message("List has an invalid name. List names must match a regular expression"));
+}
+
+// The appended exception has a purposely miswritten field (value),
+// simulating a typo or an incorrect usage.
+TEST_F(test_falco_engine, exceptions_append_no_values)
```

Comment:
Should this behavior be the same when using `overrides`? Can we test it?

Reply:
The warning is emitted also when using `override`.
Even though the underlying logic is the same I agree on adding an additional test case using `override`.

We should also change the text of the warning, since the current one can be misleading when using `override: replace`.
(`values` field is not appended but replaced with an empty one)

WDYT?

Reply:
Yes please, that additional test is appreciated